.github/workflows/pr-checks.yml

```diff
@@ -0,0 +1,19 @@
+name: PR Checks
+
+on: pull_request
```

Comment:
Originally I set this to `on: [push, pull_request]` following a guide I found on Google, but when examining [the Actions tab](https://github.com/pskfyi/handy/actions) I found that those settings were causing the checks to run multiple times! Oof. This setting instead should cause checks to run only when a PR is created, updated, or reopened after being closed.